### PR TITLE
Bug 1974241, clarified infra node sizing guidance

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1974241

Preview Build: https://deploy-preview-35298--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-host-practices?utm_source=github&utm_campaign=bot_dp#infrastructure-node-sizing_

### CODE REVIEW

File: modules/infrastructure-node-sizing.adoc [hunk 1]
```diff
@@ -29,6 +29,8 @@ The infrastructure node resource requirements depend on the cluster age, nodes,
 
 |===
 
+In general, three infrastructure nodes are recommended per worker node.
```
Comment (mohit-sheth) [inline modules/infrastructure-node-sizing.adoc:32]: In general, three infrastructure nodes are recommended per cluster.

**Review by mohit-sheth — APPROVED** [review]
other than the comment, LGTM

**Comment by ahardin-rh** [comment]
@mohit-sheth Updated. Thanks!

**Comment by ahardin-rh** [comment]
QE verified in the BZ. Merging! :rocket:

**Pull request merged**